### PR TITLE
Eliminate "3"

### PR DESCRIPTION
**Pull request by shyouhei**
Discussions: https://bugs.ruby-lang.org/issues/16803

### CODE REVIEW

**Comment by ioquatix** [comment]
If it was me, I'd literally call it `ruby/internal/file.h` to make it very clear.

**Comment by shyouhei** [comment]
@ioquatix Please leave your comment on the BTS.  Two-front discussion is hard for me.

**Comment by ioquatix** [comment]
Okay sorry, I made the same comment on RBT.

File: include/ruby/backward/2/stdalign.h [hunk 1]
```diff
@@ -20,11 +20,11 @@
  *             extension libraries. They could be written in C++98.
  * @brief      Defines #RUBY_ALIGNAS / #RUBY_ALIGNOF
```
Comment (eregon) [inline include/ruby/backward/2/stdalign.h:21]: Should be `#RBIMPL_ALIGNAS / #RBIMPL_ALIGNOF`

Comment (shyouhei) [reply include/ruby/backward/2/stdalign.h:21]: This particular header file does define `#RUBY_ALIGNAS` / `#RUBY_ALIGNOF`, which have been around for years.  We retain these macros for backward compatibility.

Comment (eregon) [reply include/ruby/backward/2/stdalign.h:21]: I misread, indeed, all fine.

**Comment by ioquatix** [comment]
Nice, love the latest commits!

**Comment by eregon** [comment]
:+1:

**Pull request merged**